armi/bookkeeping/db/compareDB3.py

```diff
@@ -493,8 +490,10 @@ def _compareComponentData(
 
         if srcSpecial ^ refSpecial:
             out.writeln(
-                "Could not compare data because one uses special formatting, "
-                "and the other does not. Ref: {} Src: {}".format(refSpecial, srcSpecial)
+                "Could not compare data for parameter {} because one uses special "
+                "formatting, and the other does not. Ref: {} Src: {}".format(
+                    paramName, refSpecial, srcSpecial
+                )
```

Comment:
This was a tiny update I made and forgot to PR. Since it's in the same file, figured I could sneak it in 😉